pkg/config/config.go

```diff
@@ -34,7 +34,7 @@ type Config struct {
 	Server       server.Config              `yaml:"server,omitempty"`
 	Metrics      metrics.Config             `yaml:"metrics,omitempty"`
 	Integrations integrations.ManagerConfig `yaml:"integrations,omitempty"`
-	Tempo        tempo.Config               `yaml:"tempo,omitempty"`
+	Traces       traces.Config              `yaml:"traces,omitempty"`
```

Comment:
Should we alias tempo-> traces like it works for loki / prometheus? It's easier to do at the root of the config, see UnmarshalYaml for how it works for the other subsystems

Reply:
Yes, just tested this and pushed the changes with unit tests